cli/src/test/java/integration/MySqlDDLIntegrationTest.java

```diff
@@ -38,11 +45,98 @@ public class MySqlDDLIntegrationTest {
 
     private static int PORT = 3306;
 
+    private static String DROP_VIEWS = "CREATE TABLE `date_time` (\n" +
+            "  `c_date` date DEFAULT NULL,\n" +
+            "  `c_year` year NOT NULL,\n" +
+            "  `c_time` time NOT NULL,\n" +
+            "  `c_timest` timestamp NOT NULL,\n" +
+            "  `c_datetime` datetime NOT NULL,\n" +
+            "  `user_date` date NOT NULL,\n" +
+            "  PRIMARY KEY (`user_date`)\n" +
+            ")";
+
+    public static String DROP_VIEWS1 = "CREATE TABLE `numeric` (\n" +
```

Comment:
Is is necessary to have the VIEWS keyword?

Reply:
No it is not necessary we can change DROP_VIEWS to something different, reason why i left it as DROP_VIEWS is because originally in postgres sql integration test this denomination was used